src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int Default_Capacity = 16;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[Default_Capacity];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > table.length * 0.75) {
```

Comment:
this threshold is calculated each time the method is used. Could it be optimized somehow?